tools/sync-translations.js

```diff
@@ -0,0 +1,98 @@
+/*!
```

Comment:
This does remove the empty translation files.

I think the 3akai-ux sync script might be simplified by using the npm crowdin module, but there are differences in how the two projects handle translations so they can't just use the same script. I.E. 3akai-ux uses properties files and needs the manifest files updated with a list of translations, ep_oae uses json files and they all just live in the locales directory.